accord-core/src/main/java/accord/api/Journal.java

```diff
@@ -42,6 +43,8 @@
  */
 public interface Journal
 {
+    Journal start(Node node);
```

Comment:
Nothing consumes the result of this function, maybe drop `Journal` in favor of `void`?  `void` makes the semantics clear where as `Journal` "could" be a different version.

<img width="477" alt="Screenshot 2025-03-27 at 3 23 11 PM" src="https://github.com/user-attachments/assets/18c62139-9cfa-47a7-b468-22de73aaab3b" />

Reply:
I’ll have to look at this again next week. I think @ifesdjeen may have said this startup sequencing issue (I forgot the precise problem) may have been addressed in another patch of his anyway.